test-workflows/pipelines/generators/clusterloader/eks/tests/1k/periodic.yaml

```diff
@@ -0,0 +1,84 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: eks-cl2-load-1k-pipeline-cron
+spec:
+  schedule: "0 8 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: eks-hello-cron
+            image: curlimages/curl
+            args: ["curl", "-X", "POST", "--data", "{}", "el-eks-cl2-load-1k-cron-listener.svc.cluster.local:8080"]
+          restartPolicy: Never
```

Comment:
Isn't this the default?

Reply:
No, as far as i know.

Reply:
Any reason we don't want to just use the default? https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#pod-template